File: OpenBots.Server.Web/ClientApp/src/app/interfaces/schedule.ts
export interface Schedule {
  agentId: string;
  agentName: string;
  // processName: string;
  automationId: string;
  automationName: string;
  createdBy: string;
  createdOn: string;
  cronExpression: string;
  deleteOn?: any;
  deletedBy?: string;
  endJobAtOccurence?: any;
  endJobOn?: any;
  expiryDate?: any;
  id: string;
  isDeleted: boolean;
  isDisabled: boolean;
  jobRecurEveryUnit?: any;
  lastExecution?: any;
  name: string;
  nextExecution?: any;
  noJobEndDate?: any;
  projectId: string;
  recurrence: boolean;
  recurrenceUnit?: any;
  startDate?: any;
  startJobOn?: any;
  startingType: string;
  status: string;
  timestamp: string;
  triggerName: string;
  updatedBy?: string;
  updatedOn?: any;
  processId?: string;
}

File: OpenBots.Server.Model/Schedule/Schedule.cs
using OpenBots.Server.Model.Core;
using System;
using System.ComponentModel;

namespace OpenBots.Server.Model
{
    public class Schedule : NamedEntity
    {
        public Guid? AgentId { get; set; }
        public Guid? AgentGroupId { get; set; }
        public string? CRONExpression { get; set; }
        public DateTime? LastExecution { get; set; }
        public DateTime? NextExecution { get; set; }
        [DefaultValue(false)]
        public bool? IsDisabled { get; set; }
        public Guid? ProjectId { get; set; }
        public string? TriggerName { get; set; }
        public bool? Recurrence { get; set; }
        public string? StartingType { get; set; }
        public DateTime? StartJobOn { get; set; }
        public DateTime? RecurrenceUnit { get; set; }
        public DateTime? JobRecurEveryUnit { get; set; }
        public DateTime? EndJobOn { get; set; }
        public DateTime? EndJobAtOccurence { get; set; }
        public DateTime? NoJobEndDate { get; set; }
        public string? Status { get; set; }
        public DateTime? ExpiryDate { get; set; }
        public DateTime? StartDate { get; set; }
        public Guid? AutomationId { get; set; }
        public Guid? QueueId { get; set; }
        [DefaultValue(0)]
        public int? MaxRunningJobs { get; set; }
    }
}
